client/src/App.js

```diff
@@ -0,0 +1,26 @@
+import React, { Component } from 'react';
+import './App.css';
+import {withRouter,Route} from 'react-router-dom';
+import SignUp from './components/signUp';
+import UserList from './components/userList';
+import SignIn from './components/signIn'
+
+class App extends Component {
+  componentDidMount=()=>{
+    console.log(this.props.history.location.pathname)
```

Comment:
after debugging always remember to remove your console.log statements. It is not a part of the standard practice.